app/server/lib/ICreate.ts

```diff
@@ -67,6 +69,11 @@ export interface ICreate {
   getStorageOptions?(name: string): ICreateStorageOptions|undefined;
   getSqliteVariant?(): SqliteVariant;
   getSandboxVariants?(): Record<string, SpawnFn>;
+
+  getLoginSystem(): Promise<GristLoginSystem>;
+
+  // Used by Grist Desktop to override the getPath function.
+  decorateDocStorageManager? (original: IDocStorageManager): void;
```

Comment:
I'm a bit concerned about having this here, with the intended purpose of overriding methods from inside an arbitrary interface. It feels like it's asking for unexpected trouble in the future.

Reply:
Refactored this in #1180 to allow a new storage manager to be injected